replacement_mode_modeling/04_FeatureClustering.ipynb

```diff
@@ -0,0 +1,2089 @@
1AmlAAAAAEidUAoAAACA1AmlAAAAAEidUAoAAACA1AmlAAAAAEidUAoAAACA1AmlAAAAAEidUAoAAACA1OU1lJo6dWr07ds3ioqKoqioKAYMGBCPPPJIbnmSJFFWVhYdO3aMgoKCGDx4cLz88st5rBgAAACAhpDXUKpTp05x3XXXxeLFi2Px4sVx2mmnxVlnnZULnq6//vq48cYbY8qUKbFo0aIoKSmJoUOHxqZNm/JZNgAAAAD1lNdQasSIEXHmmWdGjx49okePHnHttddGmzZt4tlnn40kSeLmm2+OSZMmxciRI6NPnz5x1113xdatW2PGjBn5LBsAAACAemo0z5TauXNnzJw5M7Zs2RIDBgyIFStWREVFRQwbNiy3TjabjUGDBsXChQv3up3q6uqoqqqqMQEAAADQuOQ9lFq6dGm0adMmstlsjB07Nh544IHo1atXVFRUREREcXFxjfWLi4tzy/akvLw82rZtm5s6d+58QOsHAAAAYP/lPZQ69thjY8mSJfHss8/Gt771rRg9enS88sorueWZTKbG+kmS1Br7oIkTJ8bGjRtz0+rVqw9Y7QAAAADUTct8F9CqVavo1q1bRET069cvFi1aFLfccktceeWVERFRUVERpaWlufXXrl1b6+qpD8pms5HNZg9s0QAAAADUS96vlPqwJEmiuro6unbtGiUlJTF37tzcsu3bt8f8+fNj4MCBeawQAAAAgPrK65VSV111VQwfPjw6d+4cmzZtipkzZ8a8efNi9uzZkclk4rLLLovJkydH9+7do3v37jF58uRo3bp1jBo1Kp9lAwAAAFBPeQ2l/vKXv8T5558fa9asibZt20bfvn1j9uzZMXTo0IiImDBhQmzbti3GjRsX69evj/79+8ecOXOisLAwn2UDAAAAUE95DaXuuOOOj1yeyWSirKwsysrK0ikIAAAAgFQ0umdKAQAAAND8CaUAAAAASJ1QCgAAAIDUCaUAAAAASJ1QCgAAAIDUCaUAAAAASJ1QCgAAAIDUCaUAAAAASJ1QCgAAAIDUCaUAAAAASJ1QCgAAAIDUCaUAAAAASJ1QCgAAAIDUCaUAAAAASJ1QCgAAAIDUCaUAAAAASJ1QCgAAAIDUCaUAAAAASJ1QCgAAAIDUCaUAAAAASJ1QCgAAAIDUCaUAAAAASJ1QCgAAAIDUtcx3Ac3BqlWrorKyMt9lHLSWLVuW7xIIn0M+tW/fPo466qh8lwEAALBfhFL1tGrVqujZ87jYtm1rvks56L1bvT3fJRyUtm1cFxGZOO+88/JdykGroKB1vPrqMsEUAADQpAil6qmysjK2bdsa/f/l6igqPTrf5RyU1ix9Jv7w0G2xY8eOfJdyUHp366aISOLTo66MI7v2zHc5B52qNSvjuTuvicrKSqEUAADQpAilGkhR6dHR7qhj813GQalqzcp8l0BEtOlwlH8DAAAA7DMPOgcAAAAgdUIpAAAAAFInlAIAAAAgdUIpAAAAAFInlAIAAAAgdUIpAAAAAFInlAIAAAAgdUIpAAAAAFInlAIAAAAgdUIpAAAAAFInlAIAAAAgdUIpAAAAAFInlAIAAAAgdUIpAAAAAFInlAIAAAAgdUIpAAAAAFInlAIAAAAgdUIpAAAAAFInlAIAAAAgdXkNpcrLy+Okk06KwsLC6NChQ3zlK1+J1157rcY6Y8aMiUwmU2M6+eST81QxAAAAAA0hr6HU/Pnz46KLLopnn3025s6dGzt27Ihhw4bFli1baqx3xhlnxJo1a3LTww8/nKeKAQAAAGgILfO589mzZ9eYnzZtWnTo0CGef/75OPXUU3Pj2Ww2SkpK9mmb1dXVUV1dnZuvqqpqmGIBAAAAaDCN6plSGzdujIiIdu3a1RifN29edOjQIXr06BEXXHBBrF27dq/bKC8vj7Zt2+amzp07H9CaAQAAANh/jSaUSpIkxo8fH1/4wheiT58+ufHhw4fHvffeG48//njccMMNsWjRojjttNNqXA31QRMnToyNGzfmptWrV6f1FgAAAADYR3m9fe+DLr744njppZfiqaeeqjF+zjnn5P67T58+0a9fv+jSpUv89re/jZEjR9baTjabjWw2e8DrBQAAAKDuGkUodckll8RDDz0UCxYsiE6dOn3kuqWlpdGlS5dYvnx5StUBAAAA0NDyGkolSRKXXHJJPPDAAzFv3rzo2rXrx75m3bp1sXr16igtLU2hQgAAAAAOhLw+U+qiiy6Ke+65J2bMmBGFhYVRUVERFRUVsW3btoiI2Lx5c1xxxRXxzDPPxMqVK2PevHkxYsSIaN++fZx99tn5LB0AAACAesjrlVJTp06NiIjBgwfXGJ82bVqMGTMmWrRoEUuXLo277747NmzYEKWlpTFkyJC47777orCwMA8VAwAAANAQ8n773kcpKCiIRx99NKVqAAAAAEhLXm/fAwBg/0ydOjX69u0bRUVFUVRUFAMGDIhHHnkktzxJkigrK4uOHTtGQUFBDB48OF5++eU8VgwAsGdCKQCAJqRTp05x3XXXxeLFi2Px4sVx2mmnxVlnnZULnq6//vq48cYbY8qUKbFo0aIoKSmJoUOHxqZNm/JcOQBATUIpAIAmZMSIEXHmmWdGjx49okePHnHttddGmzZt4tlnn40kSeLmm2+OSZMmxciRI6NPnz5x1113xdatW2PGjBn5Lh0AoAahFABAE7Vz586YOXNmbNmyJQYMGBArVqyIioqKGDZsWG6dbDYbgwYNioULF37ktqqrq6OqqqrGBABwIAmlAACamKVLl0abNm0im83G2LFj44EHHohevXpFRUVFREQUFxfXWL+4uDi3bG/Ky8ujbdu2ualz584HrH4AgAihFABAk3PsscfGkiVL4tlnn41vfetbMXr06HjllVdyyzOZTI31kySpNfZhEydOjI0bN+am1atXH5DaAQDe1zLfBQAAsH9atWoV3bp1i4iIfv36xaJFi+KWW26JK6+8MiIiKioqorS0NLf+2rVr
```

Comment:
This line (and the other ones like it in this file) are throwing an error for me

`ValueError: 'stat' must be one of ['count', 'density', 'probability', 'frequency'], but percent was passed.`

I wonder if it could be something like my seaborn version? I'll check what is installed against what you have in the readme

Reply:
Yep! Updating (with the instructions in the readme you included 🎉 ) fixed the issue!